app/router.ts

```diff
@@ -0,0 +1,29 @@
+import 'server-only';
+
+import { api, createRouter, route } from '@/theme-kit';
+
+export default function createAppRouter() {
```

Comment:
This router, along with the middleware, will become a part of the shared theme-kit logic that we won't be needing to reimplement in every theme.